datacite/client.py

```diff
@@ -14,61 +14,44 @@
 https://support.datacite.org/docs/mds-api-guide.
 """
 
-from __future__ import absolute_import, print_function
-
 from .errors import DataCiteError
 from .request import DataCiteRequest
 
 
 class DataCiteMDSClient(object):
     """DataCite MDS API client wrapper."""
 
-    def __init__(self, username=None, password=None, url=None, prefix=None,
-                 test_mode=False, api_ver="2", timeout=None):
+    def __init__(self, username, password, url, prefix, timeout=None):
         """Initialize the API client wrapper.
 
         :param username: DataCite username.
         :param password: DataCite password.
         :param url: DataCite API base URL. Defaults to
             https://mds.datacite.org/.
-        :param prefix: DOI prefix (or CFG_DATACITE_DOI_PREFIX). Defaults to
-            10.5072 (DataCite test prefix).
-        :param test_mode: Set to True to enable test mode. Defaults to False.
-        :param api_ver: DataCite API version. Currently has no effect.
-            Default to 2.
+        :param prefix: DOI prefix (or CFG_DATACITE_DOI_PREFIX).
         :param timeout: Connect and read timeout in seconds. Specify a tuple
             (connect, read) to specify each timeout individually.
         """
         self.username = username
         self.password = password
-        self.prefix = prefix or '10.5072'
-        self.api_ver = api_ver  # Currently not used
```

Comment:
doubt: This is currently not used, and there is no pointer that we are gonna need it in the near future? I wonder if there is a new API release we might need to revert this... However, I do prefer it out if not used, cleaner :)

Reply:
I don't see a version switch in their current API structure. Future changes could be handled with new metadata versions (which we already have) or a new constructor (like rest)